### DISCUSSION TITLE
Docker-Container mounten

### DISCUSSION THREAD

**Discussion by mrmatze007** [brutella/hkknx-public#211] [Q&A] [answered]
Hallo zusammen,

bei mir läuft hkknx in einem Docker-Container. Gerne würde ich den Pfad auf die Festplatte mounten, damit sich die Daten sauber sichern lassen (ohne manuelles Backup). Weiß zufällig jemand, wie der Mount-Pfad dazu heißt?

Vielen Dank!

Grüße,
Matthias

**Comment by mirkolenz** [comment, marked as answer]
Ich habe die App auch in Docker laufen. Der Pfad deiner Daten ist abhängig von dem Argument `-db` beim Starten von HKKNX. Ich habe bspw. `/data` gewählt. Hier ist meine Dockerfile:

```Dockerfile
FROM alpine

ARG VERSION
ARG ARCH

WORKDIR /app

RUN apk add --no-cache wget ca-certificates \
    && wget "https://github.com/brutella/hkknx-public/releases/download/${VERSION}/hkknx-${VERSION}_${ARCH}.tar.gz" \
    && tar -xzf "hkknx-${VERSION}_${ARCH}.tar.gz"

VOLUME /data

CMD [ "./hkknx", "-db", "/data", "-port", "80" ]
```

**Reply by mrmatze007** [reply]
Hi mirko,

MEGA! Danke dir für den schnellen Support. Der Tipp wird mir vermutlich noch häufiger nützlich sein ;-)

Grüße,
Matthias
